src/Kovan-DssSpell.sol

```diff
@@ -88,222 +87,211 @@ contract SpellAction {
-        // FlipperMomAbstract(FLIPPER_MOM).deny(MCD_FLIP_RENBTC_A);
+        //FlipperMomAbstract(FLIPPER_MOM).deny(MCD_FLIP_UNIV2DAIETH_A);
 
-        // Allow OsmMom to access to the RENBTC Osm
-        // !!!!!!!! Only if PIP_RENBTC = Osm and hasn't been already relied due a previous deployed ilk
-        // OsmAbstract(PIP_RENBTC).rely(OSM_MOM);
+        // Allow OsmMom to access to the UNIV2DAIETH Osm
+        // !!!!!!!! Only if PIP_UNIV2DAIETH = Osm and hasn't been already relied due a previous deployed ilk
+        LPOsmAbstract(PIP_UNIV2DAIETH).rely(OSM_MOM);
         // Whitelist Osm to read the Median data (only necessary if it is the first time the token is being added to an ilk)
-        // !!!!!!!! Only if PIP_RENBTC = Osm, its src is a Median and hasn't been already whitelisted due a previous deployed ilk
-        // MedianAbstract(OsmAbstract(PIP_RENBTC).src()).kiss(PIP_RENBTC);
+        // !!!!!!!! Only if PIP_UNIV2DAIETH = Osm, its src is a Median and hasn't been already whitelisted due a previous deployed ilk
+        MedianAbstract(LPOsmAbstract(PIP_UNIV2DAIETH).orb1()).kiss(PIP_UNIV2DAIETH);
         // Whitelist Spotter to read the Osm data (only necessary if it is the first time the token is being added to an ilk)
-        // !!!!!!!! Only if PIP_RENBTC = Osm or PIP_RENBTC = Median and hasn't been already whitelisted due a previous deployed ilk
-        // OsmAbstract(PIP_RENBTC).kiss(MCD_SPOT);
+        // !!!!!!!! Only if PIP_UNIV2DAIETH = Osm or PIP_UNIV2DAIETH = Median and hasn't been already whitelisted due a previous deployed ilk
+        LPOsmAbstract(PIP_UNIV2DAIETH).kiss(MCD_SPOT);
         // Whitelist End to read the Osm data (only necessary if it is the first time the token is being added to an ilk)
-        // !!!!!!!! Only if PIP_RENBTC = Osm or PIP_RENBTC = Median and hasn't been already whitelisted due a previous deployed ilk
-        // OsmAbstract(PIP_RENBTC).kiss(MCD_END);
-        // Set RENBTC Osm in the OsmMom for new ilk
-        // !!!!!!!! Only if PIP_RENBTC = Osm
-        OsmMomAbstract(OSM_MOM).setOsm(ILK_RENBTC_A, PIP_RENBTC);
-
-        // Set the RENBTC-A debt ceiling
-        VatAbstract(MCD_VAT).file(ILK_RENBTC_A, "line", 2 * MILLION * RAD);
-        // Set the RENBTC-A dust
-        VatAbstract(MCD_VAT).file(ILK_RENBTC_A, "dust", 100 * RAD);
+        // !!!!!!!! Only if PIP_UNIV2DAIETH = Osm or PIP_UNIV2DAIETH = Median and hasn't been already whitelisted due a previous deployed ilk
+        LPOsmAbstract(PIP_UNIV2DAIETH).kiss(MCD_END);
+        // Set UNIV2DAIETH Osm in the OsmMom for new ilk
+        // !!!!!!!! Only if PIP_UNIV2DAIETH = Osm
+        OsmMomAbstract(OSM_MOM).setOsm(ILK_UNIV2DAIETH_A, PIP_UNIV2DAIETH);
+
+        // Set the UNIV2DAIETH-A debt ceiling
+        VatAbstract(MCD_VAT).file(ILK_UNIV2DAIETH_A, "line", 3 * MILLION * RAD);
+        // Set the UNIV2DAIETH-A dust
+        VatAbstract(MCD_VAT).file(ILK_UNIV2DAIETH_A, "dust", 100 * RAD);
```

Comment:
This value is supposed to be different in Kovan, right?

Reply:
100 in Kovan, 500 in Mainnet